app/services/tweet_stat_adjustment_service.rb

```diff
@@ -0,0 +1,46 @@
+class TweetStatAdjustmentService
```

Comment:
what about `Twitter::TweetRefreshStats` ? The service name is redudant, see other namespaced services.

Reply:
Good suggestion. And I am not sure also. This service is bind with parsing the liquid nodes in article body markdown. So I am thinking, adding `TweetTag::RefreshStats` maybe?

Reply:
even better! :)

Reply:
At the end, I ended with `TweetTags::RefreshStats`, because, `TweetTag` and `Twitter` are already reserved. Please let me know if this sounds OK for you